src/components/mount-in-app/mount-in-app.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import MountInApp from '.';
+
+<Meta title="Mount In App" parameters={{ info: { disable: true }}} />
+
+# Mount In App
+Mounts a component at a specific target in the DOM. Mount In App can also be used to integrate React components into pre-existing user interfaces.
+
+## Quick Start
+Import `MountInApp` into the project.
+
+```jsx
+import MountInApp from "carbon-react/lib/components/mount-in-app";
+```
+
+### Default
+<Preview>
+  <Story name="Default">
+    {() => {
```

Comment:
This expression is not needed here, could just start with `<div>` instead of `{() => {`

Reply:
Excellent. Cheers for that. I'll get it amended.